### PR TITLE
Add a script that installs Roslyn VSIXes

### PR DESCRIPTION
**Pull request by rchande**
to the RoslynPerf hive.

### CODE REVIEW

**Comment by rchande** [comment]
cc @TyOverby @basoundr @KevinH-MS

**Comment by jasonmalinowski** [comment]
Can we unify this list with the other 10 lists we have across the codebase?

File: src/Test/Perf/util/test_util.csx [hunk 1]
```diff
@@ -61,6 +61,31 @@ class RelativeDirectory
         }
     }
 
+    public string MyBinaries()
+    {
+        // The exceptation is that scripts calling this are included
```
Comment (basoundr) [inline src/Test/Perf/util/test_util.csx]: typo `exceptation` -> `expectation`

**Comment by basoundr** [comment]
LGTM except the list @jasonmalinowski is talking about. I dont know what it is.

**Comment by jasonmalinowski** [comment]
`git grep Roslyn.VisualStudio.Setup.vsix` gives at least two sources: InstallDogfood and BuildAndTest. There might be more too.

One headache of our old perf system was it had it's own scripts for installing the product that were invariably broken every time somebody changed something else. We should endeavor not to recreate that system.

**Comment by rchande** [comment]
@jasonmalinowski The old system was based on copying over assemblies, while this script installs our vsixes. I realize that's not quite what you're hoping for, but it's still a substantial improvement.

**Comment by jasonmalinowski** [comment]
Sure, it's an improvement. It still isn't the same as what we use for regular testing, which is my concern.

**Comment by KevinH-MS** [comment]
Consider just "install.csx".  Otherwise, LGTM.

**Comment by TyOverby** [comment]
LGTM

**Pull request merged**